recipe-server/normandy/base/utils.py

```diff
@@ -82,5 +82,7 @@ def sri_hash(data):
     :param data:
         Bytes-like object containing the data to hash.
     """
-    data_hash = b64encode(sha384(data).digest())
-    return 'sha384-' + data_hash.decode()
+    data_hash = b64encode(sha384(data).digest()).decode()
+    if url_safe:
+        data_hash = data_hash.replace('+', '-').replace('/', '_')
```

Comment:
We should use [`urlsafe_b64encode`](https://docs.python.org/3.6/library/base64.html#base64.urlsafe_b64encode) instead of rolling this ourselves.

I wonder if SRI supports url-safe variants of hashes? If so, maybe we should just always use the URL safe version?